packages/bitbucket-v8-coverage-report/index.ts

```diff
@@ -47,6 +54,26 @@ function getReportResult(
   return true;
 }
 
+const getTestFailureText = (testCase: TestCase): string =>
```

Comment:
I think the index file is getting quite long. Maybe functions like getTestFailureText, getTestCaseText, and getTestCasesText could be moved to a separate utils file?